templates/case-studies/templates/_macros-quote.jinja

```diff
@@ -0,0 +1,28 @@
+# Params
+# quote: Quote text (required)
+# name: Name of quoter (required)
+# job_title: Job title of quoter (required)
+
+{%- macro quote(quote, name, job_title) -%}
```

Comment:
@jmuzina is currently working on a quote pattern and a macro for it.

https://github.com/canonical/vanilla-framework/pull/5275